tests/stdlib/os/osproc/tclose.nim

```diff
@@ -1,5 +1,6 @@
 discard """
   exitcode: 0
+  disable: osx
```

Comment:
```suggestion
  disabled: osx
```
This should fix the test failure.

Reply:
I was scouring the logs trying to spot an error and couldn't find the spec parse error. 😅 

Thanks.